src/frontend/src/app/asset-readings/chart-modal/chart-modal.component.html

```diff
@@ -7,7 +7,7 @@
           <p class="modal-card-title">{{assetCode}}</p>
         </div>
         <div class="column is-4">
-          <input-debounce placeholder="Limit(max 1000)" #limit (value)="plotReadingsGraph(assetCode, $event, offset.inputValue)" max="100"></input-debounce>
+          <input-debounce placeholder="Limit(max 1000)" #limit (value)="plotReadingsGraph(assetCode, $event, offset.inputValue)" max="1000"></input-debounce>
```

Comment:
If I am right, it should not allow user to enter Limit > 1000 and should show an error but, it allows:
![noerrorforlimit 1000](https://user-images.githubusercontent.com/29797330/33482066-51635852-d6bd-11e7-81df-2adfb7a71142.png)